mithril-common/src/digesters/immutable_digester.rs

```diff
@@ -52,9 +55,13 @@ pub enum ImmutableDigesterError {
 
     /// Error raised when there's less than the required number of completed immutables in
     /// the cardano database or even no immutable at all.
-    #[error("At least two immutables chunk should exists in directory '{db_dir}'.")]
+    #[error("At least two immutables chunk should exists in directory '{db_dir}'. Expected : {expected_number}, found : {found_number:?}.")]
```

Comment:
Maybe this is better:
`At least two immutable chunks should exist in directory '{db_dir}': expected {expected_number} but found {found_number:?}.`